### PR TITLE
Better frontend handling when API is down

### LINKED ISSUE #367: Better frontend handling when API is down
- Frontend should fail more gracefully and provide better error message when the Umbra-api is unreachable or returning error messages.
- Ideally users should still be able to perform actions relating to the base asset, i.e. sending and withdrawing Eth.

### PR DESCRIPTION
**Pull request by jferas**
Resolves #367 

Made inability to connect to Umbra API a softer error so native tokens can be transferred

### CODE REVIEW

File: frontend/src/store/wallet.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ import { ERC20_ABI, MAINNET_PROVIDER, MULTICALL_ABI, MULTICALL_ADDRESS } from 's
 import { BigNumber, Contract, ExternalProvider, Web3Provider, parseUnits } from 'src/utils/ethers';
```
Comment (garyghayrat) [inline frontend/src/store/wallet.ts:22]: _Random placement for comment_

This is actually great, thanks for picking this up. 

One nit is, could we put in a placeholder for the Token logo and Amount when API is not able to fetch those details on the receive table? And maybe handle 'LogoURI' errors a bit better?
image

Comment (jferas) [reply frontend/src/store/wallet.ts:22]: Thanks Gary! Great catch.. fixed: [ddf1367](https://github.com/ScopeLift/umbra-protocol/pull/647/commits/ddf136714f66f9f238f2c2239f4013ce75baa7b0)

**Review by alexkeating — APPROVED** [review]

File: frontend/src/store/wallet.ts (hunk 1 above)
Comment (garyghayrat) [reply frontend/src/store/wallet.ts:22]: Looks good now 
image

**Review by garyghayrat — APPROVED** [review]
lgtm!

**Pull request merged**